app/javascript/components/components/Footer.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+// import bidLogo from '../assets/bid-logo.png';
```

Comment:
Main branch should not have commented code. Make sure that it's cleaned up before you PR.